From eca904b5a6fdd478344d8c9796537dc93f25f906 Mon Sep 17 00:00:00 2001
From: Pengfei Ni <feiskyer@gmail.com>
Date: Sun, 6 Nov 2022 11:55:10 +0000
Subject: [PATCH] Fix ubuntu build

---
 src/lockdown.cpp | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/lockdown.cpp b/src/lockdown.cpp
index b95ac467..5d3890e9 100644
--- a/src/lockdown.cpp
+++ b/src/lockdown.cpp
@@ -75,11 +75,11 @@ LockdownState detect(std::unique_ptr<BPFfeature> &feature)
 {
   // Ubuntu (19.10 at least) ships a lockdown version that fully blocks the bpf
   // syscall
-  if (is_ubuntu() && !feature->has_map_array() &&
-      !feature->has_helper_probe_read())
-  {
-    return LockdownState::Confidentiality;
-  }
+  // if (is_ubuntu() && !feature->has_map_array() &&
+  //     !feature->has_helper_probe_read())
+  // {
+  //   return LockdownState::Confidentiality;
+  // }

   return read_security_lockdown();
 }
--
2.34.1
